frontend/src/pages/modelServing/screens/projects/ServingRuntimeList.tsx

```diff
@@ -26,14 +27,19 @@ const ServingRuntimeList: React.FC = () => {
   } = React.useContext(ProjectDetailsContext);
   const emptyModelServer = modelServers.length === 0;
 
+  const openModal = () => {
+    refreshServingRuntimesConfig();
```

Comment:
Why do you need to refresh before you open? It should be a relatively up to date set of data...